lib/src/text-input/TextInput.tsx

```diff
@@ -471,6 +471,7 @@ const DxcTextInput = React.forwardRef<RefType, TextInputPropsType>(
                 }}
                 backgroundType={backgroundType}
                 tabIndex={tabIndex}
+                title="Clear"
```

Comment:
I suggest being a little more explicit. Maybe something like "Clear text" would be more useful to the user.

Reply:
What do you think @aweell? Regarding the titles of the clear, increment and decrement actions, a one simple title or a more detailed one?

Reply:
I agree that we need to follow the formula: `[verb (action)] + [sustantive (the recipient of the action]`

For the input-text clear action: * "Clear field"
For the input-number stepper buttons:

* "Increment value"
* "Decrement value"

Reply:
Perfect! I change this!